src/client/driver/driver.js

```diff
@@ -92,32 +105,40 @@ const CURRENT_IFRAME_ERROR_CTORS = {
     IsInvisibleError: CurrentIframeIsInvisibleError
 };
 
+const COMMAND_EXECUTION_CHECK_DELAY = 1000;
+const COMMAND_EXECUTION_MAX_TIMEOUT = Math.pow(2, 31) - 1;
```

Comment:
May be set 2**31-1 as default value in the `waitFor` function?

Reply:
Theoretically, it's good to have a default timeout for the `waitFor` utility. But at present, for all `waitFor` call we've used a timeout specified via options from outside code. The default timeout of the `waitFor` will not be used.

Also, it's an initial implementation of the 'wait for last command result' task. It will be rewritten in a separate PR. These constants will be removed.